packages/react-pagination/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@wpmudev/react-pagination",
+  "version": "1.0.0",
+  "description": "Now I’m the model of a modern major general / The venerated Virginian veteran whose men are all / Lining up, to put me up on a pedestal / Writin’ letters to relatives / Embellishin’ my elegance and eloquence / But the elephant is in the room / The truth is in ya face when ya hear the British cannons go / BOOM",
```

Comment:
@veljkocukic please, write a correct description for this package. You can review other packages to find a good example of what to put here.